plugins/main/public/controllers/management/components/management/statistics/statistics-overview.js

```diff
@@ -237,12 +243,41 @@ export default compose(
 )(props => {
   const [loading, setLoading] = useState(false);
   const [existStatisticsIndices, setExistStatisticsIndices] = useState(false);
+  const [existStatisticsIndexPattern, setExistStatisticsIndexPattern] =
+    useState(false);
+  const indexPatternID = StatisticsDataSource.getIdentifierDataSourcePattern();
   useEffect(() => {
     const fetchData = async () => {
       try {
+        // Check the existence of related index pattern
+        const existIndexPattern = await verifyExistenceIndexPattern(
+          indexPatternID,
+        );
         setLoading(true);
-        const data = await WzRequest.genericReq('GET', '/elastic/statistics');
-        setExistStatisticsIndices(data.data);
+
+        // If the index pattern does not exist, then check the existence of index
+        if (existIndexPattern?.error?.statusCode === 404) {
```

Comment:
You can use `HTTP_STATUS_CODES.NOT_FOUND` from `plugins/main/common/constants.ts`

Reply:
Solved in https://github.com/wazuh/wazuh-dashboard-plugins/pull/7001/commits/03f9fa3df32d6f5ae61a0d2259fff878bb96290f